### ISSUE TITLE
VPR is not working with Yosys 0.19+0

### ISSUE THREAD

**Issue by KelvinChung2000** [FPGA-Research-Manchester/FABulous#79]
When running the VPR flow using Yosys 0.19+0 yields the following error. 
```
Error 1: 
Type: Blif file
File: ../nextpnr/fabulous/fab_arch/sequential_16bit_en.blif
Line: 105
Message: Failed to find matching architecture model for 'LUTFF_ESR'
```
Looks like Yosys mapping has some problem causing this error.

**Comment by gatecat** [comment]
`LUTFF_ESR` mapping shouldn't be enabled in the normal synthesis script, which one are you running?

**Comment by KelvinChung2000** [comment]
Using `./nextpnr/fabulous/fab_arch/fabulous_flow.sh`, the `.json` changed to `.blif`. 
The yosys command is the following:
`yosys -p "tcl ../synth/synth_fabulous_dffesr.tcl 4 ${1} ${1}.blif" ${1}.v -l ${1}_yosys_log.txt`

**Comment by TaoBi22** [comment]
Since we now have an upstream Yosys synthesis pass, is it worth just removing the old synthesis script and the surrounding documentation, and recommending the upstream Yosys pass instead? (As a note, based on a quick test just now I'm also getting issues with the upstream pass and VPR but I'll make another issue for that once I've explored it more).

**Comment by TaoBi22** [comment]
Never mind, the upstream pass actually seems to work fine and the issues were on my end!

**Comment by TaoBi22** [comment]
Should now be fixed in #80
